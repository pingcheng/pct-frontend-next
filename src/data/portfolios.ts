import { createPortfolio } from "@/models/Portfolio/Portfolio";

export const Portfolios = [
  createPortfolio("empire-cbs", {
    name: "Empire CBS",
    shortDescription: "Empire CBS internal job management system",
    coverImage:
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/empire-cbs/empire-cbs.png",
    url: null,
    longDescription:
      "An internal job management system that helps Empire CBS to record and track their project progress and manage their resources, timesheets and etc.",
    workplace: "Private project",
    projectRole: "Backend & DevOps Developer",
    roleDescription: [
      "Backend developments",
      "CI/CD implementation",
      "DevOps and Server management",
    ],
    members: [
      "Ping Cheng - Backend & DevOps Developer",
      "Johnson Yao - Frontend developer",
    ],
    screenshots: [
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/empire-cbs/empire-cbs-1.png",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/empire-cbs/empire-cbs-2.png",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/empire-cbs/empire-cbs-3.png",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/empire-cbs/empire-cbs-4.png",
    ],
    year: "2020-2021",
  }),
  createPortfolio("pteplus", {
    name: "PTEPLUS",
    shortDescription:
      "PTEPLUS provides a user friendly UI and accurate machine scoring for PTE test taker",
    coverImage:
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/pteplus.jpg",
    url: null,
    longDescription:
      "PTEPLUS is the first of its' kind 'smart platform' for English language practice in Australia. Its' Natural Language Processing technology provides PTE applicants with automated machine grading in alignment with the PTE Academic official score guide, such that PTE Academic future Test takers may practise for the Test, and also easily gauge their current standards and Test-readiness.",
    workplace: "Eduplus 2016 - 2019",
    projectRole: "Development Lead",
    roleDescription: [
      "The decision on tech stack",
      "Architecture design and implement",
      "Requirements analysis and breakdown",
      "Tasks split up and allocation",
      "Backend/Frontend developments",
      "Bridging different teams",
    ],
    members: [
      "Ping Cheng - Development lead",
      "Zoe Wang - Frontend developer",
      "Wayne Zhang - Backend developer",
      "Jeffery Wu - Backend developer",
      "Irene Zhang - Frontend developer",
    ],
    screenshots: [
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/2.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/aio1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/aio2.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/aio3.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/aio4.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/exam1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/op1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/op2.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/op3.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/op4.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/op5.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pteplus/report-analysis.jpg",
    ],
    year: "2016-2019",
  }),
  createPortfolio("pinhr", {
    name: "PinHR",
    shortDescription: "A customised human resources management tool",
    coverImage:
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/cover.jpg",
    url: null,
    longDescription:
      "A customised human resources management tool, including online application form and auto email system",
    workplace: "EDUPLUS",
    projectRole: "Sole Developer",
    roleDescription: [
      "The decision on tech stack",
      "Architecture design and implement",
      "Requirements analysis and breakdown",
      "Backend/Frontend developments",
    ],
    members: ["Ping Cheng - Developer"],
    screenshots: [
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/2.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/3.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/4.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/5.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/6.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/pinhr/7.jpg",
    ],
    year: "2018",
  }),
  createPortfolio("vatgc", {
    name: "VAGTC",
    shortDescription: "The final year project in the university",
    coverImage:
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/vagtc/vagtc.png",
    url: "https://www.vagtc.org.au/",
    longDescription:
      "The final year project in the university, worked with the non-profit organisation to provide a modern and clean website. The web is WordPress based.",
    workplace: "Swinburne University",
    projectRole: "Developer",
    roleDescription: [
      "Setup the WordPress",
      "Customise the theme",
      "Integrate plugins",
    ],
    members: [
      "Fame - Team Leader",
      "Ping Cheng - Developer",
      "Salil Batra - Developer",
    ],
    screenshots: [
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/vagtc/1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/vagtc/2.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/vagtc/3.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/vagtc/4.jpg",
    ],
    year: "2015",
  }),
  createPortfolio("qce-eduplus", {
    name: "QCE Eduplus",
    shortDescription:
      "Maths practice platform partners with Cengage for Queensland secondary schools",
    coverImage:
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/cover.jpg",
    url: null,
    longDescription:
      "EduPlus empowers teachers and students with immediate feedback, live progress tracking, and real-time actionable insights â€“ so you can harness more potential with less effort",
    workplace: "EDUPLUS",
    projectRole: "Development Lead",
    roleDescription: [
      "The decision on tech stack",
      "Architecture design and implement",
      "Requirements analysis and breakdown",
      "Tasks split up and allocation",
      "Backend/Frontend developments",
      "Bridging different teams",
    ],
    members: [
      "Ping Cheng - Development Lead",
      "Jeffery Wu - Backend/Frontend developer",
    ],
    screenshots: [
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/2.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/3.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/4.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/5.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/6.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/7.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/qce-eduplus/8.jpg",
    ],
    year: "2019-2020",
  }),
  createPortfolio("rhfoto", {
    name: "RH.foto",
    shortDescription:
      "A simplistic and clean photo wall for a group of professional photographers",
    coverImage:
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/rhfoto/rhfoto.png",
    url: "https://rhfoto.com.au/",
    longDescription:
      "A simple and clean Wordpress based website for a group of professional wedding photographers demonstrates their abilities to deliver high end, breathtaking, fashionable and personalised images.",
    workplace: "Private project",
    projectRole: "Sole Developer",
    roleDescription: [
      "Wordpress customisation",
      "Theme development",
      "Plugin integration",
    ],
    members: ["Ping Cheng - Developer"],
    screenshots: [
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/rhfoto/1.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/rhfoto/2.jpg",
      "https://pct-frontend-assets.s3.ap-southeast-2.amazonaws.com/portfolio/rhfoto/3.jpg",
    ],
    year: "2017",
  }),
];
