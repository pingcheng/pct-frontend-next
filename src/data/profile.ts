export const profile = {
  fullName: "Ping Cheng",
} as const;

export const socialUrls = {
  github: "https://github.com/pingcheng",
} as const;
