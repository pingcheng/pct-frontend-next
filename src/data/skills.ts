export const backendStack: string[] = [
  "Node, TypeScript, JavaScript",
  "Java, ColdFusion",
  "PHP, Laravel, Moodle",
  "MySQL, MSSQL, Redis",
  "Server management, Message Queue",
  "Payment Integration",
];

export const frontendStack: string[] = [
  "HTML",
  "JavaScript, Vue, React, jQuery",
  "CSS, SASS, TailwindCSS, Styled Components, Bootstrap",
  "Webpack",
];

export const devOpsStack: string[] = [
  "AWS",
  "Infrastructure planning and implementing",
  "Horizontal scaling",
  "Serverless, Docker, ECS",
  "CI/CD, GitlabCI, CircleCI",
];
