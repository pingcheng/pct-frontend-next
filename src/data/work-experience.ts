export type WorkPosition = {
  position: string;
  startDate: string;
  endDate: string;
  description: string[];
};

export type Company = {
  name: string;
  linkedInUrl?: string;
};

export type WorkExperience = {
  company: Company;
  positions: WorkPosition[];
};

export const workExperiences: WorkExperience[] = [
  {
    company: {
      name: "REA Group - PropTrack",
      linkedInUrl: "https://www.linkedin.com/company/rea-group",
    },
    positions: [
      {
        position: "Lead Developer",
        startDate: "Aug 2024",
        endDate: "Present",
        description: [],
      },
      {
        position: "Senior Developer",
        startDate: "Jul 2022",
        endDate: "Aug 2024",
        description: [
          "Highly involved in digital mortgage areas, seamlessly connecting banks and the valuation industry with modern digital solutions. Contributing to the development of secure, enterprise-level products.",
          ":line-break:",
          "Key Responsibilities:",
          "- Collaborate closely with the lead developer to design system solutions.",
          "- Utilise existing group-wide infrastructure to optimise efficiency and avoid redundancy.",
          "- Take the lead in developing feature slices and efficiently allocate work within the team.",
          "- Provide mentorship to less experienced developers within the team.",
        ],
      },
      {
        position: "Developer",
        startDate: "Aug 2021",
        endDate: "Jun 2022",
        description: [],
      },
    ],
  },
  {
    company: {
      name: "AroFlo",
      linkedInUrl: "https://www.linkedin.com/company/aroflo",
    },
    positions: [
      {
        position: "Web Developer",
        startDate: "Aug 2019",
        endDate: "Sept 2021",
        description: [
          "Working as a full stack developer. Delivering new features to AroFlo Platform which is an industry leading product in Australia, my current main involves in the platform include but not limited to project management, quotes, invoices, payment integration areas.",
        ],
      },
    ],
  },
  {
    company: {
      name: "PTEPLUS",
    },
    positions: [
      {
        position: "Development Lead",
        startDate: "Jan 2018",
        endDate: "May 2019",
        description: [
          "Work with the dev team on two main products. Design and implement IT architecture on the clouding computing platforms (AWS and Aliyun). Deliver stable, rapid responsible and reliable web applications to product's users.",
        ],
      },
      {
        position: "Full Stack Developer",
        startDate: "Oct 2016",
        endDate: "Jan 2018",
        description: [
          "One of the key contributors to the start-up company, and participated in almost every stage of product design, implement and testing. Took responsibilities on full solutions to the technical side such as coding and servers' setup.",
        ],
      },
    ],
  },
];

export const backendStack: string[] = [
  "Node, TypeScript, JavaScript",
  "Java, ColdFusion",
  "PHP, Laravel, Moodle",
  "MySQL, MSSQL, Redis",
  "Server management, Message Queue",
  "Payment Integration",
];

export const frontendStack: string[] = [
  "HTML",
  "JavaScript, Vue, React, jQuery",
  "CSS, SASS, TailwindCSS, Styled Components, Bootstrap",
  "Webpack",
];

export const devOpsStack: string[] = [
  "AWS",
  "Infrastructure planning and implementing",
  "Horizontal scaling",
  "Serverless, Docker, ECS",
  "CI/CD, GitlabCI, CircleCI",
];
