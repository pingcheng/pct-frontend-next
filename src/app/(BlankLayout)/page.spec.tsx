import { render, screen, within } from "@testing-library/react";
import Home from "./page";
import { profile, socialUrls } from "@/data/profile";

describe("test home page", () => {
  it("ensure snapshot is correct", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  it("should display the avatar with correct dimensions", () => {
    render(<Home />);
    // The avatar should be in the document
    const avatar = document.querySelector("img");
    expect(avatar).toBeInTheDocument();
    expect(avatar).toHaveAttribute("width", "192");
    expect(avatar).toHaveAttribute("height", "192");
  });

  it("should display the correct name from profile data", () => {
    render(<Home />);
    // Test that the profile name appears in the heading
    const heading = screen.getByRole("heading", { level: 1 });
    expect(heading).toHaveTextContent(profile.fullName);
  });

  it("should display description text with correct styling", () => {
    render(<Home />);
    // Test for new description text content
    expect(screen.getByText(/Staff Engineer/)).toBeInTheDocument();
    expect(screen.getByText(/latest tech/)).toBeInTheDocument();

    // Test that key technical skills are mentioned and highlighted
    ["Node", "TypeScript", "PHP", "Java", "Linux"].forEach((skill) => {
      const el = screen.getByText(skill, { exact: false });
      expect(el).toBeInTheDocument();
      expect(el).toHaveClass("text-primary");
    });
  });

  it("should render all necessary social links with correct URLs", () => {
    render(<Home />);

    // Check for GitHub link with correct URL
    const githubLink = screen.getByText("Github").closest("a");
    expect(githubLink).toBeInTheDocument();
    expect(githubLink).toHaveAttribute("href", socialUrls.github);

    // Check for LinkedIn link with correct URL
    const linkedinLink = screen.getByText("LinkedIn").closest("a");
    expect(linkedinLink).toBeInTheDocument();
    expect(linkedinLink).toHaveAttribute("href", socialUrls.linkedin);

    // Check for internal links
    const portfolioLink = screen.getByText("Portfolio").closest("a");
    expect(portfolioLink).toBeInTheDocument();
    expect(portfolioLink).toHaveAttribute("href", "/portfolio");

    const aboutLink = screen.getByText("About").closest("a");
    expect(aboutLink).toBeInTheDocument();
    expect(aboutLink).toHaveAttribute("href", "/about");
  });

  it("should render social links with their corresponding icons", () => {
    render(<Home />);

    // The social links section should contain SVG icons
    const socialLinksSection = screen.getByText("Github").closest("div");
    expect(socialLinksSection).toBeInTheDocument();

    // Each social link should have an SVG icon
    const links = screen.getAllByText(/(Github|LinkedIn|Portfolio|About)/);
    links.forEach((link) => {
      const linkElement = link.closest("a");
      const svg = linkElement?.querySelector("svg");
      expect(svg).toBeInTheDocument();
    });
  });

  it("should have proper layout structure", () => {
    render(<Home />);

    // Check main container
    const main = screen.getByRole("main");
    expect(main).toHaveClass(
      "h-screen",
      "flex",
      "flex-col",
      "items-center",
      "justify-center"
    );

    // Check that elements are in the correct order: Avatar, Description, Social Links
    const elements = main.children;
    expect(elements.length).toBeGreaterThanOrEqual(3);

    // First element should contain the avatar
    expect(elements[0].querySelector("img")).toBeInTheDocument();

    // Second element should contain the description with profile name
    expect(elements[1].textContent).toContain(profile.fullName);

    // Third element should contain the social links
    expect(elements[2].textContent).toContain("Github");
    expect(elements[2].textContent).toContain("LinkedIn");
  });
});
