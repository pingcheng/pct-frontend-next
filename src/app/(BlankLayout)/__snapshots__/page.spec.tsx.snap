// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`test home page ensure snapshot is correct 1`] = `
<div>
  <script
    type="application/ld+json"
  >
    {"@context":"https://schema.org","@type":"Person","name":"Ping Cheng","jobTitle":"Lead Developer","url":"https://www.pingchengtech.com","image":"https://www.pingchengtech.com/apple-icon.png","sameAs":["https://github.com/pingcheng","https://www.linkedin.com/in/ping-cheng-5a47b484"],"worksFor":{"@type":"Organization","name":"REA Group","url":"https://www.rea-group.com"},"knowsAbout":["Node.js","TypeScript","JavaScript","PHP","Java","Linux","React","Vue.js","AWS","Docker","MySQL","Redis"],"address":{"@type":"PostalAddress","addressLocality":"Melbourne","addressRegion":"Victoria","addressCountry":"Australia"}}
  </script>
  <script
    type="application/ld+json"
  >
    {"@context":"https://schema.org","@type":"WebSite","name":"Ping Cheng Tech","url":"https://www.pingchengtech.com","description":"Personal website and portfolio of Ping Cheng, Lead Developer and Software Engineer in Melbourne","author":{"@type":"Person","name":"Ping Cheng","url":"https://www.pingchengtech.com"}}
  </script>
  <main
    class="h-screen flex flex-col items-center justify-center mx-auto gap-8 page-container"
  >
    <section
      class="animate-fade-in-scale"
    >
      <div
        class="max-w-full "
        style="width: 192px; height: 192px;"
      >
        <img
          alt="Ping Cheng profile photo"
          data-nimg="1"
          decoding="async"
          height="192"
          src="http://localhost/_next/image?url=https%3A%2F%2Favatars0.githubusercontent.com%2Fu%2F8447539&w=384&q=75"
          srcset="/_next/image?url=https%3A%2F%2Favatars0.githubusercontent.com%2Fu%2F8447539&w=256&q=75 1x, /_next/image?url=https%3A%2F%2Favatars0.githubusercontent.com%2Fu%2F8447539&w=384&q=75 2x"
          style="color: transparent; border-radius: 50%;"
          width="192"
        />
      </div>
    </section>
    <section
      class="description animate-fade-in animate-delay-200"
    >
      <h1>
        Hey, Iâ€™m 
        <span
          class="text-primary"
        >
          Ping Cheng
        </span>
        .
      </h1>
      <p>
        <span
          class="text-primary"
        >
          Lead dev
        </span>
        , code lover, and always building something cool with the
        <span
          class="text-primary"
        >
           latest tech
        </span>
        .
      </p>
      <p>
        Into 
        <span
          class="text-primary"
        >
          Node
        </span>
        ,
         
        <span
          class="text-primary"
        >
          TypeScript
        </span>
        ,
         
        <span
          class="text-primary"
        >
          PHP
        </span>
        ,
         
        <span
          class="text-primary"
        >
          Java
        </span>
        ,
         
        <span
          class="text-primary"
        >
          Linux
        </span>
        , and making things work better.
      </p>
    </section>
    <nav
      aria-label="Social links and navigation"
      class="flex flex-wrap content-around text-lg justify-center animate-fade-in animate-delay-400"
    >
      <div
        class="animate-fade-in animate-delay-500"
      >
        <a
          aria-label="Github (opens in new tab)"
          class="socialLink"
          href="https://github.com/pingcheng"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            aria-hidden="true"
          >
            <svg
              fill="none"
              height="1em"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
              />
            </svg>
          </span>
           
          Github
        </a>
      </div>
      <div
        class="animate-fade-in animate-delay-600"
      >
        <a
          aria-label="LinkedIn (opens in new tab)"
          class="socialLink"
          href="https://www.linkedin.com/in/ping-cheng-5a47b484"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            aria-hidden="true"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 512 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M444.17 32H70.28C49.85 32 32 46.7 32 66.89v374.72C32 461.91 49.85 480 70.28 480h373.78c20.54 0 35.94-18.21 35.94-38.39V66.89C480.12 46.7 464.6 32 444.17 32zm-273.3 373.43h-64.18V205.88h64.18zM141 175.54h-.46c-20.54 0-33.84-15.29-33.84-34.43 0-19.49 13.65-34.42 34.65-34.42s33.85 14.82 34.31 34.42c-.01 19.14-13.31 34.43-34.66 34.43zm264.43 229.89h-64.18V296.32c0-26.14-9.34-44-32.56-44-17.74 0-28.24 12-32.91 23.69-1.75 4.2-2.22 9.92-2.22 15.76v113.66h-64.18V205.88h64.18v27.77c9.34-13.3 23.93-32.44 57.88-32.44 42.13 0 74 27.77 74 87.64z"
              />
            </svg>
          </span>
           
          LinkedIn
        </a>
      </div>
      <div
        class="animate-fade-in animate-delay-700"
      >
        <a
          aria-label="Portfolio"
          class="socialLink"
          href="/portfolio"
        >
          <span
            aria-hidden="true"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 13a1.5 1.5 0 0 0 1.5 1.5h13A1.5 1.5 0 0 0 16 13V6a1.5 1.5 0 0 0-1.5-1.5h-13A1.5 1.5 0 0 0 0 6zM2 3a.5.5 0 0 0 .5.5h11a.5.5 0 0 0 0-1h-11A.5.5 0 0 0 2 3m2-2a.5.5 0 0 0 .5.5h7a.5.5 0 0 0 0-1h-7A.5.5 0 0 0 4 1"
              />
            </svg>
          </span>
           
          Portfolio
        </a>
      </div>
      <div
        class="animate-fade-in animate-delay-800"
      >
        <a
          aria-label="About"
          class="socialLink"
          href="/about"
        >
          <span
            aria-hidden="true"
          >
            <svg
              fill="none"
              height="1em"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="12"
                cy="12"
                r="10"
              />
              <path
                d="M8 14s1.5 2 4 2 4-2 4-2"
              />
              <line
                x1="9"
                x2="9.01"
                y1="9"
                y2="9"
              />
              <line
                x1="15"
                x2="15.01"
                y1="9"
                y2="9"
              />
            </svg>
          </span>
           
          About
        </a>
      </div>
    </nav>
  </main>
</div>
`;
